server/application/auth/passwords.py

```diff
@@ -0,0 +1,17 @@
+import secrets
+
+
+class PasswordEncoder:
+    def hash(self, value: str) -> str:
+        raise NotImplementedError  # pragma: no cover
+
+    def verify(self, password: str, hash: str) -> bool:
+        raise NotADirectoryError  # pragma: no cover
```

Comment:
Why the `NotADirectoryError`? Is it a bad auto-complete?

Reply:
En effet !